primitives/state-machine/src/overlayed_changes/mod.rs

```diff
@@ -629,6 +632,40 @@ impl OverlayedChanges {
 				overlay.next_change(key)
 			)
 	}
+
+	/// Set a value in the offchain storage.
+	#[cfg(feature = "std")]
+	pub fn offchain_set_storage(&mut self, prefix: &[u8], key: &[u8], value: &[u8]) {
+		self.offchain.set(prefix, key, value);
+	}
+
+	/// Clear a value in the offchain storage.
+	#[cfg(feature = "std")]
+	pub fn offchain_remove_storage(&mut self, prefix: &[u8], key: &[u8]) {
+		self.offchain.remove(prefix, key);
+	}
+
+	/// Get a value in the offchain storage.
+	#[cfg(feature = "std")]
+	pub fn offchain_get_storage(
```

Comment:
I don't think this will ever be needed, the API is append-only. But fine to keep it for completeness.